labs/RivePlayer/src/RivePlayer.cs

```diff
@@ -0,0 +1,287 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using RiveSharp;
+using SkiaSharp;
+using System.Collections.Concurrent;
+using System.Net.Http;
+using Windows.Storage.Streams;
+using Windows.Storage;
+
+namespace CommunityToolkit.Labs.WinUI.Rive;
+
+// This is a high level XAML control for playing a Rive state machine or animation. The state
```

Comment:
Should look at [xmldoc](https://docs.microsoft.com/dotnet/csharp/language-reference/xmldoc/) comments later as they'll better bubble up in VS intellisense.

Reply:
Ack, will do!